st2common/tests/unit/test_purge_worklows.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Extreme Networks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from datetime import timedelta
+
+from st2common import log as logging
+from st2common.constants.triggers import TRIGGER_INSTANCE_PROCESSED
```

Comment:
As TRIGGER_INSTANCE_PROCESSED is unused it will break the CI checks, so can you remove this import.